scripts/west_commands/twister_cmd.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2022 Google
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+
+from west import log
+
+from zephyr_ext_common import Forceable
+from zephyr_ext_common import get_twister_path
+
+# Resolve path to twister libs and add imports
+twister_path = get_twister_path()
+os.environ["ZEPHYR_BASE"] = str(twister_path.parent)
+
+sys.path.insert(0, str(twister_path))
+sys.path.insert(0, str(twister_path / "pylib" / "twister"))
+
+from importlib.util import spec_from_loader, module_from_spec
+from importlib.machinery import SourceFileLoader
+
+spec = spec_from_loader(
+    "twister", SourceFileLoader("twister", str(twister_path / "twister"))
+)
+twister = module_from_spec(spec)
+spec.loader.exec_module(twister)
+
+from twisterlib.environment import add_parse_arguments, parse_arguments
+
+BUILD_USAGE = """\
+west twister [-h]
+"""
+
+BUILD_DESCRIPTION = f"""\
```

Comment:
Unnecessary f-string; the linter may warn on this.

Reply:
done